libraries/platform/iot_network.h

```diff
@@ -94,6 +111,178 @@ typedef void ( * IotNetworkReceiveCallback_t )( IotNetworkConnection_t pConnecti
                                                 void * pContext );
 /* @[declare_platform_network_receivecallback] */
 
+/**
+ * @brief Provide an asynchronous notification of network closing
+ *
+ * A function with this signature may be set with @ref platform_network_function_setclosecallback
+ * to be invoked when data is available on the network.
+ *
+ * @param[in] pConnection The connection on which data is available, defined by
+ * the network stack.
+ * @param[in] reason The reason the connection was closed
+ * @param[in] pContext The third argument passed to @ref platform_network_function_setreceivecallback.
```

Comment:
should be platform_network_function_setclosecallback().

Reply:
good catch, fixed